index.js

```diff
@@ -0,0 +1,44 @@
+const form = document.getElementById('form'); // Store html form element to variable
+const bookList = document.getElementById('bookList'); // store html ul element to var
+
+let booksList = JSON.parse(localStorage.getItem('books')) || []; // get local storage of books saved and pass it to object or array if false
+
+function addBook(title, author) { // Gets parameter, pushes to booksList and saves to local storage
+  const newBook = { title, author };
+  booksList.push(newBook);
+  localStorage.setItem('books', JSON.stringify(booksList));
+  /* eslint-disable no-use-before-define */
+  displayBooks(); // Call display books to update list
+}
```

Comment:
- [OPTIONAL] 🤔 Kindly Prevent the user for submitting a form if the Author and Title input field are empty. You may use the `required` html input attribut or write your own javaScript validation. 👍🏻 

- [OPTIONAL] Great job so far 👏🏻  However I would suggest using array methods that don't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice`, and so on. (line: 8 ). Immutable array methods write Cleaner JavaScript Code. For more documentation you can visit the this [How to Write Remarkably Clean JavaScript Code](https://www.sitepoint.com/immutable-array-methods-javascript/)